### ISSUE TITLE
wazuh-packages is broken for Arch Linux packages

### ISSUE THREAD

**Issue by avielw** [wazuh/wazuh-packages#1649]
Since a few versions ago, the script to create packages for Arch (generate_arch_package.sh) is broken. I've created a fix and I'll create a pull request.

**Comment by rauldpm** [comment]
Hello @avielw!

Thank you very much for opening the PR to us, indeed, we are aware that the generation of arch packages is broken right now, in fact, I opened this issue https://github.com/wazuh/wazuh-packages/issues/1273 to fix it.

In principle, regarding the PR, we have to review it since from 4.2 to 4.3 certain things have changed that must be taken into account and that we must test exhaustively, such as the upgrade due to the change of user and group from ossec to wazuh. In principle, I leave you an initial review on the PR

- https://github.com/wazuh/wazuh-packages/pull/1650#pullrequestreview-1004596478

A comment in the PR about files not modified but that would need to be changed:

- https://github.com/wazuh/wazuh-packages/pull/1650#issuecomment-1154146736

We note this PR in our roadmap for a most exhaustive review and pertinent tests, thank you very much for your contribution! 

Regards, Raúl.

**Comment by avielw** [comment]
I saw in #1273 that the package generation does not work on 4.2.5.
Is it really an issue? That version didn't have a support for Arch anyway (The agent and server code for Arch were merged into version 4.3 and is one of the features in its release notes).

**Comment by rauldpm** [comment]
You are right, I have reviewed your comments to the review and I have left you another review, please take a look as there may be a problem with the uninstall process https://github.com/wazuh/wazuh-packages/pull/1650#pullrequestreview-1012181760

Thanks!
